### PR TITLE
Operating cost documentation updates

### PR DESCRIPTION
**Pull request by kdayday**
Branch for cost documentation updates (documentation only, no changes to struct fields)
- There are changes to OperationalCost getter/setters where they were either missing or the docstring was wrong
- OperationalCost getter/setters are exported where they were missing
- Add operating cost how-to
- Also bug fixes pictures in the documentation weren't showing up due to pointing to wrong location
- Couple other minor docstring bug fixes outside costs

### CODE REVIEW

File: docs/src/tutorials/add_cost_curve.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Following the decision steps above:
+1. We select [`RenewableGenerationCost`](@ref) to represent this renewable generator.
+2. We select a [`LinearCurve`](@ref) to represent the \$22/MWh variable cost.
+
+Following the implementation steps, we define `RenewableGenerationCost` by nesting the definitions:
+```@repl costcurve
+RenewableGenerationCost(variable = CostCurve(value_curve = LinearCurve(22.0)))
+```
+
+!!! tip
```
Comment (kdayday) [inline docs/src/tutorials/add_cost_curve.md:47]: kwdef tip here

Comment (kdayday) [reply docs/src/tutorials/add_cost_curve.md]: Removed along with docstring change to remove kwdef

**Comment by kdayday** [comment]
Outstanding items:
- @GabrielKS 's updates to the cost table including graphics of each operating cost
- (me and @GabrielKS) Moving the ValueCurve and FuelCurve/CostCurve structs up a level in the Model Library so each has its own page and all getter/setters visible

@jd-lara @rodrigomha  In case I'm out sick later this week,  I think it would be beneficial to merge this branch before releasing PSY4 even if those  items^ aren't resolved yet.

**Comment by GabrielKS** [comment]
> @jd-lara @rodrigomha In case I'm out sick later this week, I think it would be beneficial to merge this branch before releasing PSY4 even if those items^ aren't resolved yet.

Agreed — especially since given recent conversations with @jd-lara @rodrigomha I'll be modifying the cost structs themselves a bit, can we get this reviewed and merged now and then I'll add my documentation pieces on a subsequent PR?

**Review by jd-lara — APPROVED** [review]

**Pull request merged**